app/services/invoice_rebilling_initialise.service.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+/**
+ * @module InvoiceRebillingInitialiseService
+ */
+
+const { InvoiceModel } = require('../models')
+const { RebillingInvoicesPresenter } = require('../presenters')
+
+class InvoiceRebillingInitialiseService {
+  /**
+   * Creates two empty invoices on a given bill run for rebilling purposes: a 'cancel' invoice and a 'rebill' invoice.
+   * The cancel invoice is the invoice which will cancel out the invoice being rebilled, and the rebill invoice is the
+   * one that will rebill the the customer. They can be identifed by rebillType `C` and `R` respectively. Both invoices
+   * will be linked to the original invoice by rebilledInvoiceId, which contains the id of the original invoice.
+   *
+   * @param {module:BillRunModel} billRun An instance of `BillRunModel` of the bill run to add the invoices to.
+   * @param {module:InvoiceModel} invoice An instance of `InvoiceModel` of the invoice to be rebilled.
+   *
+   * @returns {Object} A reponse object containing `cancelInvoice`, `rebillInvoice` and `response`.
+   * @returns {module:InvoiceModel} `resonse.cancelInvoice` The cancel invoice.
+   * @returns {module:InvoiceModel} `response.rebillInvoice` The rebill invoice.
+   * @returns {Array} `response.response` An array containing the cancel invoice id and type `C`, and the rebill invoice
+   * id and type `R`.
+   */
+  static async go (billRun, invoice) {
+    const cancelInvoice = await this._createInvoice(billRun, invoice, 'C')
```

Comment:
It might be prudent to create both invoices inside a transaction. Then should one fail both fail and it'll be a little easier to recover from.

Reply:
Good shout 👍 Done!